package.json

```diff
@@ -17,55 +17,59 @@
     "test-vite:watch": "vitest --watch",
     "test-storybook": "test-storybook --browsers firefox",
     "test-storybook:coverage": "npm run test-storybook -- --coverage",
-    "lint": "prettier --plugin-search-dir . --check . && eslint .",
-    "format": "prettier --plugin-search-dir . --write .",
+    "test-storybook:ci": "concurrently -k -s first -n \"SB,TEST\" -c \"magenta,blue\" \"npx http-server storybook-static --port 6006 --silent\" \"npx wait-on tcp:127.0.0.1:6006 && npm run test-storybook:coverage\"",
```

Comment:
I'm a bit lazy so can you please explain this part? `-k -s first -n \"SB,TEST\" -c \"magenta,blue\"` I'm curious 🥺

Reply:
It's just for styling, it prepends `SB` and `TEST` before each concurrent part and also adds some colors to it :sweat_smile: